mpls-lab/conn_test.yaml

```diff
@@ -0,0 +1,14 @@
+# connectivity tests from netaautomatorr to
+#But no IDea so sar...I think we need to get the Container IP or execute the chnge via Telnet ?
+- hosts: localhost
+  gather_facts: no
+  vars_files:
+    - inventory/gns3_vars.yml
+  collections:
+    - davidban77.gns3
+
+- hosts: netautomator-1
+  gathet_facts: no
+
+  tasks:
+    - name: ping test
```

Comment:
This can be done through telnet and console to the device yes... but that means that your script needs to be in `pexpect`...

The other way is through the `netautomator`, but for that you need an SSH/Ansible connection to it... So for now you can just delete this file

Reply:
removed from the repo